style.css

```diff
@@ -0,0 +1,183 @@
+/* Fonts used */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+/* CSS reset */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+/* Main styles */
+body {
+  font-family: 'Poppins', sans-serif;
+  height: 100vh;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  padding: 1rem;
+  height: 100%;
+}
+
+header {
+  padding: 0 8rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.logo-text {
+  color: #6070ff;
+}
+
+.hidden {
+  display: none;
+  color: #6070ff;
+}
+
+.menu {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu-item {
+  margin-right: 2rem;
+}
+
+.headline {
+  display: flex;
+  align-items: center;
+  background-image: url(img/Header-bg.svg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 100%;
+  background-color: #fff;
+  height: calc(100% - 1rem);
+}
+
+.main {
+  margin: 0 auto;
+  width: 65%;
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  font-size: 48px;
+  line-height: 60px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.main-text {
+  font-size: 20px;
+  line-height: 28px;
+  width: 85%;
+  color: #344563;
+}
+
+.connect-text {
+  font-size: 1rem;
+  line-height: 24px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-top: 1rem;
+}
+
+.icon-list {
+  display: flex;
+}
+
+li {
+  list-style: none;
+}
+
+.fab {
+  font-size: 20px;
+  margin-right: 1.2rem;
+  margin-top: 0.6rem;
+  color: #505f79;
+}
+
+/* Media queries */
+@media (max-width: 768px) {
+  header {
+    padding: 0;
+  }
+
+  .menu-item {
+    margin-right: 1rem;
+  }
+
+  .main {
+    width: 100%;
+  }
+
+  .main-text {
+    width: 100%;
+    margin-top: 0.8rem;
+    font-size: 1rem;
+  }
+}
+
+@media (max-width: 375px) {
```

Comment:
[ OPTIONAL ] Please do not use media queries here as it's the mobile-first design approach.